githooks/bash/create_checksums.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+CHECKSUM_DIRECTORY="$PWD/checksums"
```

Comment:
Can precommit change the working dir?

Reply:
If you mean as configuration setting per hook, I don't think so.